Formula/libtomcrypt.rb

```diff
@@ -0,0 +1,29 @@
+class Libtomcrypt < Formula
+  desc "Comprehensive, modular and portable cryptographic toolkit"
+  homepage "http://www.libtom.net"
+  url "https://github.com/libtom/libtomcrypt/archive/v1.18.0.tar.gz"
+  sha256 "bdba1499dab3bf3365fa75553f069eba7bea392e8f9e0381aa0e950a08bd85ba"
+
+  option "with-gmp", "enable gmp as MPI provider"
+  option "with-libtommath", "enable libtommath as MPI provider"
+
+  depends_on "gmp" => :optional
+  depends_on "libtommath" => :optional
+
+  def install
+    ENV.append "CFLAGS", "-DUSE_GMP -DGMP_DESC" if build.with?("gmp")
```

Comment:
```ruby
if build.with? "gmp"
  ENV.append "CFLAGS", "-DUSE_GMP -DGMP_DESC"
  ENV.append "EXTRALIBS", "-lgmp"
end
if build.with? "libtommath"
  ENV.append "CFLAGS", "-DUSE_LTM -DLTM_DESC"
  ENV.append "EXTRALIBS", "-ltommath"
end
```
or such. Although how you've set this out makes me think you want to enforce either/or rather than allowing both?

Reply:
Both should be allowed.

Reply:
Does it actually link to both simultaneously in a useful manner?

Reply:
> It is possible to build the library against all MPI providers in parallel and choose at startup-time which math library should be used.